ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import {
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+    ActionsColumn,
+    IAction,
+} from '@patternfly/react-table';
+import { format } from 'date-fns';
+
+import usePagination from 'hooks/patternfly/usePagination';
+import useTableSort from 'hooks/patternfly/useTableSort';
+
+import { ComplianceScanResultsOverview } from './types';
```

Comment:
If any of these correspond to response types, let’s remember to import from service file in the future.